bgpd/bgp_route.c

```diff
@@ -11037,22 +11053,19 @@ DEFUN (show_ip_bgp_regexp,
 				 bgp_show_type_regexp, uj);
 }
 
-DEFUN (show_ip_bgp_instance_all,
-       show_ip_bgp_instance_all_cmd,
-       "show [ip] bgp <view|vrf> all ["BGP_AFI_CMD_STR" ["BGP_SAFI_WITH_LABEL_CMD_STR"]] [json]",
-       SHOW_STR
-       IP_STR
-       BGP_STR
-       BGP_INSTANCE_ALL_HELP_STR
-       BGP_AFI_HELP_STR
-       BGP_SAFI_WITH_LABEL_HELP_STR
-       JSON_STR)
+DEFUN(show_ip_bgp_instance_all, show_ip_bgp_instance_all_cmd,
```

Comment:
Here as well, please don't change this the whole formatting. We should do this with a separate PR.